### PR TITLE
A2C

### PR DESCRIPTION
**Pull request by belerico**
## Summary

Describe the purpose of the pull request, including:

- What problem does it solve?
- How does it solve the problem?
- What are the benefits of the solution?
- Any relevant background or context information.
- Reference any relevant issues or pull requests.

This PR solves #32. The additional tasks needed are:

- [x] A2C coupled algorithm
- [ ] A2C decoupled algorithm
- [ ] A2C pixel-based
- [ ] A2C recurrent 

## Checklist

Please confirm that the following tasks have been completed:

- [x] I have tested my changes locally and they work as expected. (Please describe the tests you performed.)
- [x] I have added unit tests for my changes, or updated existing tests if necessary.
- [x] I have updated the documentation, if applicable.
- [x] I have installed pre-commit and run locally for my code changes.

## Screenshots or Visuals (Optional)

If applicable, please provide screenshots, diagrams, graphs, or videos of the changes, features or the error.

## Additional Information (Optional)

Please provide any additional information that may be useful for the reviewer, such as:

- Any potential risks or challenges associated with the changes.
- Any instructions for testing or running the code.
- Any other relevant information.

Thank you for your contribution! Once you have filled out this template, please ensure that you have assigned the appropriate reviewers and that all tests have passed.

### CODE REVIEW

File: sheeprl/configs/exp/ppo_torchrl.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# @package _global_
```
Comment (michele-milesi) [inline sheeprl/configs/exp/ppo_torchrl.yaml:1]: What config is this?

File: sheeprl/configs/algo/ppo_torchrl.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+defaults:
```
Comment (michele-milesi) [inline sheeprl/configs/algo/ppo_torchrl.yaml:1]: What config is this?

File: sheeprl/algos/a2c/args.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
```
Comment (michele-milesi) [inline sheeprl/algos/a2c/args.py:1]: do you need this file? Or can we delete it?

File: sheeprl/algos/a2c/utils.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict
+
+import torch
+from lightning import Fabric
+
+from sheeprl.algos.a2c.agent import A2CAgent
+from sheeprl.utils.env import make_env
+
+AGGREGATOR_KEYS = {"Rewards/rew_avg", "Game/ep_len_avg", "Loss/value_loss", "Loss/policy_loss"}
```
Comment (michele-milesi) [inline sheeprl/algos/a2c/utils.py:9]: Can you please add the `MODELS_TO_REGISTER` variable? And also the configs in the `./sheeprl/configs/model_manager` folder.
Thanks

File: sheeprl/configs/algo/ppo_torchrl.yaml (hunk 2 above)
Comment (belerico) [reply sheeprl/configs/algo/ppo_torchrl.yaml]: Something that has slipped through the PR: I've deleted it

File: sheeprl/configs/exp/a2c.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# @package _global_
+
+defaults:
+  - override /algo: a2c
+  - override /env: gym
```
Comment (michele-milesi) [inline sheeprl/configs/exp/a2c.yaml:5]: Can you add `- override /model_manager: a2c?
Thanks

File: sheeprl/algos/a2c/args.py (hunk 3 above)
Comment (michele-milesi) [inline sheeprl/algos/a2c/args.py:1]: Can you please delete this file?
Thanks

Comment (belerico) [reply sheeprl/algos/a2c/args.py]: My eyes have gone blind! Done!

**Review by michele-milesi — APPROVED** [review]

**Pull request merged**